tests/order/order.variables.js

```diff
@@ -34,144 +27,254 @@ const newOrder = {
   },
   items: [
     {
-      category: [
-        {
-          lang: 'uk',
-          value: 'Сумки',
-        },
-        {
-          lang: 'en',
-          value: 'Bags',
-        },
-      ],
-      subcategory: [
-        {
-          lang: 'uk',
-          value: 'Сумки',
-        },
-        {
-          lang: 'en',
-          value: 'Bags',
-        },
-      ],
-      model: [
-        {
-          lang: 'uk',
-          value: 'Сумка з гобеленом',
-        },
-        {
-          lang: 'en',
-          value: 'Bag with a Pattern',
-        },
-      ],
-      name: [
-        {
-          lang: 'uk',
-          value: 'Сумка з гобеленом синя',
-        },
-        {
-          lang: 'en',
-          value: 'Bag with a Pattern Blue',
-        },
-      ],
-      colors: [
-        [
-          {
-            lang: 'uk',
-            value: 'Сталево-блакитний',
-          },
-          {
-            lang: 'en',
-            value: 'Steel-blue',
-          },
-        ],
-      ],
-      pattern: [
-        {
-          lang: 'uk',
-          value: 'Олені',
-        },
-        {
-          lang: 'en',
-          value: 'Deers',
-        },
-      ],
-      closure: [],
-      closureColor: '',
-      size: {
-        heightInCm: 38,
-        widthInCm: 36,
-        depthInCm: 10,
-        volumeInLiters: 0,
-        weightInKg: 0,
+      product: productId,
+      model: modelId,
+      quantity: 2,
+      isFromConstructor: false,
+      constructorBasics: constructorId,
+      options: {
+        size: sizeId,
       },
-      bottomMaterial: [
+      fixedPrice: [
         {
-          lang: 'uk',
-          value: 'Тканина Кордура',
+          currency: 'UAH',
+          value: 7000,
         },
         {
-          lang: 'en',
-          value: 'Cordura fabric',
+          currency: 'USD',
+          value: 240,
         },
       ],
-      bottomColor: [
-        {
-          lang: 'uk',
-          value: 'чорний',
-        },
-        {
-          lang: 'en',
-          value: 'black',
-        },
+    },
+  ],
+  paymentStatus: 'CREATED',
+});
+const newSizeInputData = {
```

Comment:
I think it would be better to move this file func in separate file inside size folder